src/sentry/notifications/manager.py

```diff
@@ -251,14 +252,20 @@ def get_for_users_by_parent(
     def get_for_recipient_by_parent(
         self,
         type: NotificationSettingTypes,
-        parent: Union["Organization", "Project"],
-        recipient: "User",
+        parent: Union["Organization", "Project", "Team"],
```

Comment:
Team is not a parent type.

Reply:
Why not? The scope identifier for this type of db row is the team id

Reply:
The parent is the parent in parent-specific notification settings. Parent-independent notification setting have scope type `USER` or `TEAM`.